### PR TITLE
Bump version to v0.4

### PR DESCRIPTION
**Pull request by javadamiri**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+0.4.0 (2021-05-17)
+===
+
+* Fixes a bug where benchmarks got stock randomly due to a synchronisation issue
+* Removes duplicate edges
```
Comment (qinsoon) [inline CHANGELOG.md:5]: It is actually a bug fix that caused duplicated edges.

**Review by qinsoon — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (javadamiri) [reply CHANGELOG.md:5]: Do you mean duplicate edges caused the synchronisation bug?

Comment (qinsoon) [reply CHANGELOG.md:5]: We push roots multiple times that caused the duplicate edges, which is a bug. This is what I meant.

Comment (javadamiri) [reply CHANGELOG.md:5]: Right, I will change to reflect that

**Pull request merged**